modules/zenith-public/lua/4-additive/quests/repeatableQuestDoubleRewards.lua

```diff
@@ -0,0 +1,664 @@
+-----------------------------------
+-- Repeatable Quest Double Gil Rewards Module
+-- Doubles gil rewards for whitelisted repeatable quests
+-- Limited to first 3 completions per day PER QUEST
+-----------------------------------
+require('modules/module_utils')
+
+local m = Module:new('repeatableQuestDoubleRewards')
+
+-----------------------------------
+-- Configuration
+-----------------------------------
+local maxDailyBonuses = 3
+
+-----------------------------------
+-- Module-level context storage (keyed by player ID)
+-- Used to track which quest is being turned in
+-----------------------------------
+local playerQuestContext = {}
+
+-----------------------------------
+-- Lazy-loaded whitelist of repeatable quests
+-- Using numeric IDs to avoid load-order issues
+-----------------------------------
+local repeatableQuestsCache = nil
+
+local function getRepeatableQuests()
+    if repeatableQuestsCache then
+        return repeatableQuestsCache
+    end
+
+    repeatableQuestsCache =
+    {
+        -- San d'Oria Quests (questLog = 0)
+        [0] =
+        {
+            [78] = true,  -- FEAR_OF_THE_DARK
```

Comment:
use enums and the set util: 

```suggestion
        [xi.questLog.SANDORIA] =
        set{
            xi.quest.id.sandoria.FEAR_OF_THE_DARK,
...
```

the `set` util is defined in `scripts/utils/common.lua` and just generates a table as you are doing, but without the chance of typos

Reply:
done